src/main/java/models/Ticket.java

```diff
@@ -1,12 +1,14 @@
 package models;
 
+import Interface.InterfacePrintInformation;
+import services.AutoCreateID;
 import services.TicketService;
 
 import java.time.LocalDateTime;
 
-public class Ticket {
+public class Ticket extends AutoCreateID implements InterfacePrintInformation {
```

Comment:
add shared(arg1) and shared(arg1, arg2) methods - item 4 in homework task

Reply:
According this, i create method "typeDeviceAccordingEventCode". And This code play this function.

Reply:
how does it replace overloading?